Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
-name = "lal"
-version = "3.7.0"
 authors = ["Eirik Albrigtsen <ealbrigt@cisco.com>"]
 description = "A strict, language-agnostic build system and dependency manager"
 license = "MIT"
+name = "lal"
+version = "3.7.0"
```

Comment:
alphabetical keys?

Reply:
ugh, it's just different tools keep re-ordering them. both clippy and cargo-edit change the key order in different ways. maybe one of them parses into a hashmap rather than a btreemap.